index.html

```diff
@@ -7,8 +7,11 @@
     <link rel="icon" type="image/png" href="../photos/MJTsignature.png">
     <link rel="stylesheet" href="/style/style.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/font-mfizz/font-mfizz.css">
+
     <!-- <link href="https://fonts.googleapis.com/css?family=Palanquin" rel="stylesheet"> -->
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
+    <script src='https://www.google.com/recaptcha/api.js'></script>
```

Comment:
I tried but it didn't work. I will look further into this possibility in the next ticket (the one concerned with the recaptcha)